assets/js/presets.js

```diff
@@ -1,13 +1,39 @@
 const Presets = {
-  default: {
```

Comment:
Hier fehlen jetzt die presets. Schau mal in die _config.yml unter cloundinary. Dort sind verschiedene Presets definiert und diese sollten ja hier abgebildet werden, so dass ich die 1:1 in vue nutzen kann. Die anderen Parameter die intern als "defaults" sind, sollte es natürlich auch geben, aber hier sind jetzt 2 Dinge vermischt, die defaults und die presets.